src/main/java/edu/ucsb/cs156/happiercows/controllers/UserCommonsController.java

```diff
@@ -49,47 +49,46 @@ public class UserCommonsController extends ApiController {
   @PreAuthorize("hasRole('ROLE_ADMIN')")
   @GetMapping("")
   public UserCommons getUserCommonsById(
-      @ApiParam("userId") @RequestParam Long userId,
-      @ApiParam("commonsId") @RequestParam Long commonsId) throws JsonProcessingException {
-
-    UserCommons userCommons = userCommonsRepository.findByCommonsIdAndUserId(commonsId, userId)
-        .orElseThrow(
-            () -> new EntityNotFoundException(UserCommons.class, "commonsId", commonsId, "userId", userId));
-    return userCommons;
-  }
+    @ApiParam("userId") @RequestParam Long userId,
```

Comment:
In general, please try to avoid making changes that only affect the formatting of a file without changing the code, unless you have a good reason to. This makes the formatting/spacing of the affected file different from that of the rest of the project. Was this an intentional change? If not, consider reverting it.

Reply:
It was intentional, and almost all the changes I made to formatting were for the purpose of making the formatting and spacing consistent. For the file I changed the most, I changed it because I was having trouble determining the scope of objects due to the poor spacing.